pkg/apis/app/v2/kieapp_types.go

```diff
@@ -152,18 +152,31 @@ type KieServerSet struct {
 	KafkaJbpmEventEmitters *KafkaJBPMEventEmittersObject `json:"kafkaJbpmEventEmitters,omitempty"`
 	Cors                   *CORSFiltersObject            `json:"cors,omitempty"`
 	// MDBMaxSession number of KIE Executor sessions
-	MDBMaxSession *int `json:"MDBMaxSession,omitempty"`
+	MDBMaxSession    *int              `json:"MDBMaxSession,omitempty"`
+	TerminationRoute *TerminationRoute `json:"terminationRoute,omitempty"`
+}
+
+type TerminationRoute struct {
+	// +kubebuilder:validation:Enum:=http;https
+	// Protocol to use, this couldmoverride the global disableSSL, http is the default, otherwise https
+	Protocol string `json:"protocol,omitempty"`
+	// +kubebuilder:validation:Enum:=passthrough;edge
+	// Passthrough is the default protocol, otherwise edge could be used.
+	Termination string `json:"termination,omitempty"`
+	// Enable the tls section when http port is used
```

Comment:
when https is used?

Reply:
when https is used the enabletls is true by default, we need to enable this section with http and edge termination, prior to this change wasn't possible

Reply:
see the suggestion on the same line.